wgsl/index.bs

```diff
@@ -5826,16 +5826,140 @@ these functions must only be invoked in uniform control flow.
 
 ### Arrayed resource access TODO ### {#arrayed-resource-access}
 
-## Floating Point Evaluation TODO ## {#floating-point-evaluation}
+## Floating Point Evaluation ## {#floating-point-evaluation}
+
+[SHORTNAME] follows the IEEE 754 standard for floating point computation with
+the following exceptions:
+* No floating point exceptions are generated.
+* Signaling NaNs may not be generated.
+    Any signaling NaN may be converted to a quiet NaN.
+* Implementations may assume that NaNs, inifinities are not present
+    * Note: This means some functions (e.g. `isInf`, `isNan`, `min` and `max`)
+        may not return the expected result due to optimizations about the presence
+        of NaNs and infinities.
+* Implementations may ignore the sign of a zero.
+    That is, a zero with a positive sign may behave like a zero a with a negative sign, and vice versa.
+* No rounding mode is specified.
+* Implementations may flush denormalized value on the input and/or output of
+    any operation listed in [[#floating-point-accuracy]].
+    * Other operations are required to preserve denormalized numbers.
+* The accuracy of operations is given in [[#floating-point-accuracy]].
+
+### Floating Point Accuracy ### {#floating-point-accuracy}
+
+An operation is <dfn noexport>correctly rounded</dfn> if its result is a
+floating point number nearest to the exact result computed with infinite
+precision.
+The rounding mode is unspecified for [SHORTNAME].
+If the result, `x`, is exactly representable in floating point, then `x` must be
+returned.
+Otherwise, a floating point value closest to `x`, either no less than `x`
+or no greater than `x` must be returned.
+
+The units in the last place, <dfn noexport>ULP</dfn>, for a floating point
+number `x` is the minimum distance between two non-equal floating point numbers
```

Comment:
"The units ... is the minimum". Should this be singular "The unit"?

Reply:
Right. Could be.